flask-server/main.py

```diff
@@ -229,7 +228,32 @@ def check_denylist(token):
     if token_query:
         return True
     else:
-        return False
+        return False        
+
+@app.route('/apis/add-bookmark', methods=['POST'])
```

Comment:
Can add in the future but also we should validate user before adding bookmarks

Reply:
But in theory if verify the jwt when the user goes to the page, do we have verify again when in the api